app/src/main/java/me/quadphase/qpdex/Party.java

```diff
@@ -0,0 +1,94 @@
+package me.quadphase.qpdex;
+
+//
+//
+//  Generated by StarUML(tm) Java Add-In
+//
+//  @ Project : QPdex (C) 2015
+//  @ File Name : Party.java
+//  @ Date : 21-Jul-15
+//  @ Author : Nicole
+//
+//
+
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * These <code>Pokemon</code> are in your party. They each have a particular set of moves.
+ */
+public class Party {
+
+    /**
+     * Max: 6
+     */
+    private List<Pokemon> pokemon;
+
+    /**
+     * 1 move set per Pokemon in the <code>pokemon</code> list
+     */
+    private List<MoveSet> moveSets;
+
+    /**
+     * Constructor to make an empty party
+     */
+    public Party() {
```

Comment:
That's the good thing of having a normalized database: we'd need to modify it, but it doesn't break our original scheme. I think we might have to discuss multi party stuff further, but it can be done and it isn't hard. We just won't make this a singleton when implementing methods in the next issue...